test/client/actions/admin_contactlist.test.js

```diff
@@ -42,6 +42,33 @@ describe('Add Campaign actions', () => {
     });
   });
 
+  describe('setCurrentContactList', () => {
+    const setCurrentContactListResult = setCurrentContactList(contactListFixture);
```

Comment:
Or just unpack it here to be: 
`const { type, payload } = setCurrentContactList(contactListFixture)`